src/ciphers/aes-gcm.browser.ts

```diff
@@ -29,10 +45,15 @@ export function create (opts?: CreateOptions): AESCipher {
       password = fromString(password)
     }
 
+    let cryptoKey
```

Comment:
Prevents accidental assignment of an unrelated type in the future.

```suggestion
    let cryptoKey: CryptoKey
```